general/networking/apache.md

```diff
@@ -0,0 +1,58 @@
+---
+uid: network-reverse-proxy-apache
+title: Apache Reverse Proxy
+---
+
+## Apache HTTP Server Project
+
+"The [Apache HTTP Server Project](https://httpd.apache.org/) is an effort to develop and maintain an open-source HTTP server for modern operating systems including UNIX and Windows. The goal of this project is to provide a secure, efficient and extensible server that provides HTTP services in sync with the current HTTP standards." - Apache Home Page
```

Comment:
I would format this in a code block or something, maybe just moving the `- Apache Home Page` to a new line would make it nicer.

Reply:
moving it to a new line invoked weird formatting. I just took it out.